plugins/my-plugin-two/plugin/src/Log.ts

```
@@ -0,0 +1,9 @@
+export class Log {
+  static log(str: string) {
+    console.log(`\texpo-notification-service-extension-plugin: ${str}`)
+  }
+
+  static error(str: string) {
+    console.error(`\texpo-notification-service-extension-plugin: ${str}`)
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)